packages/lesswrong/components/vulcan-core/App.tsx

```diff
@@ -40,18 +40,18 @@ const App = ({serverRequestStatus, timeOverride, history}: ExternalProps & {
   const locale = localeSetting.get();
 
   useEffect(() => {
-    void userChangedCallback.runCallbacks({
-      iterator: currentUser,
-      properties: [],
-    });
+    if (!bundleIsServer) {
+      const { onUserChanged } = require('@/client/logging');
```

Comment:
In cases like this, I think it would be better to encapsulate the `bundleIsServer` check, i.e. put `onUserChanged` somewhere like `'@/lib/logging'` and have it return a noop if `bundleIsServer` is true. It might not be clear to future devs that this is preventing `@/client/logging` from being included in the server bundle, not just ensuring it doesn't run, so they could easily fail to do this check properly when copying it to somewhere else